ui/src/components/LandingPage.css

```diff
@@ -0,0 +1,5 @@
+.title {
+    padding-top: .5em;
+    margin-bottom: .5em;
+    margin-left: 2em;
```

Comment:
```suggestion
    margin-left: 1em;
```
With `2em` the title looks too indented - `1em` aligns the header text, title text, and table text, wdyt?

Reply:
When I use 1em I see this: 

<img width="324" alt="Screenshot 2019-03-12 at 11 55 44" src="https://user-images.githubusercontent.com/6498727/54198230-d7b54680-44bd-11e9-8af7-ce9cef2f4c5c.png">

Could you post a screenshot also?

Reply:
![Screenshot from 2019-03-12 11-59-53](https://user-images.githubusercontent.com/12085451/54198454-6e820300-44be-11e9-9900-2a0e369e483b.png)
![Screenshot from 2019-03-12 12-00-06](https://user-images.githubusercontent.com/12085451/54198457-72158a00-44be-11e9-8e4d-a32c6e12ec7d.png)

Reply:
With `2em`:
![Screenshot from 2019-03-12 12-02-53](https://user-images.githubusercontent.com/12085451/54198597-d3d5f400-44be-11e9-803d-73ff157443f2.png)
![Screenshot from 2019-03-12 12-03-09](https://user-images.githubusercontent.com/12085451/54198599-d6384e00-44be-11e9-8d26-19101dd0cc39.png)

Reply:
![Screenshot from 2019-03-12 12-04-10](https://user-images.githubusercontent.com/12085451/54198636-f536e000-44be-11e9-91a9-0cf251098a9d.png)

Reply:
I'm using Chrome: Version 72.0.3626.109. Strange that I have exactly the same and it looks very different. Would the version of Chrome make that much difference?

Reply:
@damienomurchu pushed a new change which should resolve this. Would you mind verifying?